### PR TITLE
Data loading refactor

### PR DESCRIPTION
**Pull request by mfalkiewicz**
#38 Refactor of data loading functions. I also refactored the configuration parameters. Instead of a single dict inside object, we have separate fields for each parameter that can be easily changed after object initialization.

### CODE REVIEW

File: TotalActivation/TotalActivation.py [hunk 1]
```diff
@@ -10,33 +10,60 @@
...
+
+    def _load_data(self, f, a=None, mask=True, type='nifti', detrend=True, standardize=True, highpass=0.01, lowpass=None,
+             TR=2):
+        """
+        Wrapper function for loading all kinds of data
+        
+        :return: data or data + atlas in 2D 
+        """
+
+        global cmd
```
Comment (kjurek) [inline TotalActivation/TotalActivation.py]: remove global

File: TotalActivation/TotalActivation.py [hunk 2]
```diff
@@ -10,33 +10,60 @@
...
-        self.n_tp = 0
-        self.cost_save = False
-        self.masking = False
+
+    def _load_data(self, f, a=None, mask=True, type='nifti', detrend=True, standardize=True, highpass=0.01, lowpass=None,
+             TR=2):
+        """
+        Wrapper function for loading all kinds of data
+        
+        :return: data or data + atlas in 2D
```
Comment (kjurek) [inline TotalActivation/TotalActivation.py]: this function does not return anything so this comment seems to be misleading

File: TotalActivation/TotalActivation.py [hunk 3]
```diff
@@ -100,15 +131,15 @@ def _deconvolve(self):
         :return:
         """
 
-        if self.config['Method_space'] == None:
+        if self.method_space == None:
```
Comment (kjurek) [inline TotalActivation/TotalActivation.py]: is None

File: TotalActivation/TotalActivation.py [hunk 4]
```diff
@@ -10,33 +10,60 @@
...
+    def _load_data(self, f, a=None, mask=True, type='nifti', detrend=True, standardize=True, highpass=0.01, lowpass=None,
+             TR=2):
+        """
+        Wrapper function for loading all kinds of data
+        
+        :return: data or data + atlas in 2D 
+        """
+
+        global cmd
+        if type is 'nifti':
```
Comment (kjurek) [inline TotalActivation/TotalActivation.py]: type is a python keyword, I would change name of this variable to something else like ftype

File: TotalActivation/TotalActivation.py [hunk 5]
```diff
@@ -100,15 +131,15 @@ def _deconvolve(self):
         :return:
         """
 
-        if self.config['Method_space'] == None:
+        if self.method_space == None:
             print("Temporal regularization...")
```
Comment (kjurek) [inline TotalActivation/TotalActivation.py]: replace prints with logger

**Review by kjurek — APPROVED** [review]

File: TotalActivation/TotalActivation.py (hunk 5 above)
Comment (mfalkiewicz) [reply TotalActivation/TotalActivation.py]: What's the advantage?

**Pull request merged**